ocs_ci/ocs/ui/mcg_ui.py

```diff
@@ -370,3 +384,57 @@ def check_obc_option(self, text="Object Bucket Claims"):
         namespace_obj = OCP(kind=constants.NAMESPACE, namespace=sc_name)
         namespaces.append(namespace_obj)
         delete_projects(namespaces)
+
+
+class ObUi(BucketsUi):
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    def delete_ob_ui(self, delete_via, expect_fail):
+        """
+        Delete an OBC via the UI
+
+        delete_via (str): delete via 'OB/Actions' or via 'three dots'
+        expect_fail (str): verify if OB removal fails with proper PopUp message
+        """
+        self.navigate_object_buckets_page()
+        self.delete_resource(delete_via, "ObjectBucket")
+
+        if expect_fail:
+
+            def _check_three_dots_disabled(text):
+                logger.info(text)
+                # when three_dots element is active attribute 'disabled' does not exist
+                assert (
+                    self.get_element_attribute(
+                        self.generic_locators["three_dots"], "disabled"
+                    )
+                    == "true"
+                ), "Three dots are active after OBC removal and OB removal action. Expected - disabled"
+
+                # PopUp is not reachable via Selenium driver. It does not appear in DOM
+                import requests
+                import json
```

Comment:
should we move these imports to top of the file?

Reply:
not sure it's right. We use **requests & json** only here and it's called only in this method, so why to import it in any occasion when something else needed from the file

Reply:
I agree, keeping it at the top makes it more readable.

Reply:
done